From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Mon, 31 Mar 2025 21:06:56 +0200
Subject: [PATCH] HACK: Disable Tracker tests

The tracker tests are broken in our containers because bubblewrap
stopped working, also see https://bugs.archlinux.org/task/78033
---
 tests/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index 026f2a360395..34028851981c 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -40,5 +40,4 @@ if local_metadata_enabled and lua_factory_enabled
 endif
 
 if get_option('enable-tracker3') != 'no' and tracker3_dep.found()
-    subdir('tracker3')
 endif
